tests/test_inner.py

```diff
@@ -779,9 +828,13 @@ def test_workflow_docker_build_error():
                                                   }}],
                                    plugin_files=[this_file])
 
-    assert workflow.build_docker_image().is_failed()
-
+    try:
+        workflow.build_docker_image()
+    except Exception as ex:
+        pass
```

Comment:
I had that before, but Lui pointed out that the asserts after won't be run anymore, which is what I need to check